decidim-proposals/lib/decidim/proposals/component.rb

```diff
@@ -265,7 +265,20 @@
         visibility: "all"
       ) do
         proposal = Decidim::Proposals::Proposal.new(params)
+
         proposal.add_coauthor(participatory_space.organization)
```

Comment:
```suggestion
```
Shouldn't we drop this line, as it's already covered by line 279 (else clause)? Or is there something that I'm missing?